lib/builder.js

```diff
@@ -3,7 +3,7 @@ const themeServer =
 const fs = require('fs');
 const request = require('superagent');
 const chalk = require('chalk');
-const renderHtml = require('./render-html');
+const renderHtml = require('./render-html').default;
```

Comment:
This project uses a mix of CommonJS and ES6 modules.  This should probably be made consistent to prevent bugs like this.
I won't address it in this PR.